frontend/packages/pipelines-plugin/src/components/pipelines/detail-page-tabs/EventListenerDetails.tsx

```diff
@@ -11,6 +11,7 @@ export interface EventListenerDetailsProps {
 
 const EventListenerDetails: React.FC<EventListenerDetailsProps> = ({ obj: eventListener }) => {
   const { t } = useTranslation();
+  const triggers = eventListener.spec.triggers?.filter((trigger) => trigger?.template?.name) || [];
```

Comment:
nit: 
```suggestion
  const triggers = eventListener.spec.triggers?.filter((trigger) => trigger.template?.name) || [];
```

Reply:
I saw that too @invincibleJai... I didn't call it out because technically the item could be `null` in an array... but it's probably unlikely. Still probably best to drop the optionality for ease of reading all the same 👍